src/antelope/stores/utils/index.ts

```diff
@@ -14,8 +14,40 @@ const REVERT_PANIC_SELECTOR = '0x4e487b71';
 
 export const WEI_PRECISION = 18;
 
-export function formatWei(bn: BigNumber, tokenDecimals: number, displayDecimals = 4): string {
-    const amount = BigNumber.from(bn);
+/**
+ * divideBn performs a division of two float numbers represented as strings or native numbers.
+ * @param a is the numerator expressed as a number or a string representing a number (e.g. '100000000002', '1.5' or '0.000000000000000001')
+ * @param b is the denominator expressed as a number or a string representing a number
+ * @returns a string representing the result of the division also as a float number
+ */
+export function divideBn(a: string | number, b: string | number): string {
```

Comment:
i think this and `multipleBn` should be renamed to use `float`, like `divideFloat`. the name currently sounds like it divides a `BigNumber` type

Reply:
agree